tests/backup/backup_namespace_labelled_test.go

```diff
@@ -1505,3 +1506,187 @@ var _ = Describe("{ManualAndScheduleBackupUsingNamespaceLabel}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// NamespaceLabelledBackupOfEmptyNamespace takes namespace labelled backup of empty namespace and restores it
+var _ = Describe("{NamespaceLabelledBackupOfEmptyNamespace}", func() {
+	var (
+		backupLocationUID           string
+		cloudCredUID                string
+		clusterUid                  string
+		credName                    string
+		backupLocationName          string
+		labelForNamespace           string
+		periodicSchedulePolicyName  string
+		periodicSchedulePolicyUid   string
+		scheduleName                string
+		firstScheduleBackupName     string
+		emptyNamespaceRestore       string
+		nextScheduleBackupName      string
+		scheduleBackupWithNamespace string
+		restoreWithNamespaces       string
+		schPolicyInterval           int64
+		cloudCredUidList            []string
+		bkpNamespaces               []string
+		scheduleBackupList          []string
+		nsLabels                    map[string]string
+		scheduledAppContexts        []*scheduler.Context
+	)
+	backupLocationMap := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+	schPolicyInterval = 15
+	JustBeforeEach(func() {
+		StartTorpedoTest("NamespaceLabelledBackupOfEmptyNamespace", "NamespaceLabelledBackupOfEmptyNamespace takes namespace labelled backup of empty namespace and restores it", nil, 86693)
+		log.InfoD("Deploy applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < 3; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts := ScheduleApplications(taskName)
```

Comment:
Is it not supposed to be core.Instance().CreateNamespace?

Reply:
Discussed. Hence resolving